ovs/extensions/healthcheck/alba_hc.py

```diff
@@ -284,6 +289,14 @@ def check_if_proxies_work(result_handler):
                                     break
                                 if time.time() - namespace_delete_start > AlbaHealthCheck.NAMESPACE_TIMEOUT:
                                     raise RuntimeError('Delete namespace has timed out after {0}s'.format(time.time() - namespace_start_time))
+
+                            # be tidy, and make the proxy forget the namespace
+                            try:
+                                AlbaCLI.run(command='proxy-statistics',
+                                            named_params=proxy_named_params,
+                                            extra_params=['--forget', namespace_name])
+                            except:
+                                result_handler.warning('failed to make proxy forget namespace {0}.'.format(namespace_name)
```

Comment:
Also change 'failed' to 'Failed'. These are messages the user will see so we would like to be formal :)